src/components/Login.js

```diff
@@ -0,0 +1,222 @@
+/* eslint-disable no-undef */
+import {initializeApp} from "firebase/app";
+import React, {useState} from "react";
+import {useNavigate} from "react-router-dom";
+import {
+  getAuth,
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import {Button} from "../styles/Profile.module";
+import {getFirestore, doc, setDoc, onSnapshot} from "firebase/firestore";
+import {LoginWrapper} from "../styles/Login.module";
+
+const firebaseConfig = {
+  apiKey: process.env.REACT_APP_FIREBASE_APIKEY,
+  authDomain: process.env.REACT_APP_FIREBASE_DOMAIN,
+  projectId: process.env.REACT_APP_FIREBASE_PROJECTID,
+  storageBucket: process.env.REACT_APP_FIREBASE_STORAGEBUCKET,
+  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGINGSENDERID,
+  appId: process.env.REACT_APP_FIREBASE_APPID,
+  measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENTID,
+};
+
+// Initialize Firebase
+const app = initializeApp(firebaseConfig);
+const auth = getAuth();
+const db = getFirestore(app);
+
+function Login(props) {
+  const SIGN_IN = "sign_in";
+  const SIGN_UP = "sign_up";
```

Comment:
Normally, we will keep constant's string as camel-style as well. And keep the variance constant name as snake-style(use '_') since we need to make constant variance all capital so that we cannot use camel-style.
E.g. const SIGN_IN = "signUp";

Reply:
And please at least put constant out of Login.

Reply:
Have them arranged in const.js.